scss/_patterns_search-and-filter.scss

```diff
@@ -0,0 +1,168 @@
+@import 'settings';
+
+@mixin vf-p-search-and-filter {
+  @include vf-search-and-filter;
+  @include vf-filter-panel-section;
+}
+
+@mixin vf-search-and-filter {
+  .p-search-and-filter {
+    $height: 2.5rem;
+
+    border: 1px solid $color-mid-dark;
+    position: relative;
+
+    .p-search-and-filter__search-container {
+      align-items: center;
+      background-color: $color-x-light;
+      display: flex;
+      flex-wrap: wrap;
+      height: auto;
+      margin: 0;
+      min-height: $height;
+      overflow: hidden;
+      padding: 0 2rem 0 0.5rem;
+      position: relative;
+
+      &[data-active='true'] {
+        height: auto;
+      }
+
+      &[data-empty='false'],
+      &[aria-expanded='false'] {
+        height: $height;
+      }
+
+      .p-chip {
+        margin-bottom: 0.5rem;
+      }
+
+      .p-search-and-filter__selected-count {
+        color: $color-information;
+        cursor: pointer;
+        position: absolute;
+        right: 0.5rem;
+        top: 6px;
+      }
+
+      &[aria-expanded='true'] {
+        height: auto;
+        max-height: 100%;
+
+        .p-search-and-filter__selected-count {
+          display: none;
+        }
+      }
+
+      .p-search-box {
+        flex-grow: 1;
+      }
+    }
+
+    .p-search-and-filter__panel {
+      background: $color-x-light;
+      border-radius: $border-radius;
+      box-shadow: $box-shadow;
+      opacity: 1;
+      padding: 0.5rem 1rem 0;
+      position: absolute;
+      transition: opacity 0.25s;
+      width: 100%;
+      z-index: 9999;
+
+      &[aria-hidden='true'] {
+        opacity: 0;
+        pointer-events: none;
+      }
+    }
+
+    .p-search-prompt {
+      background: $color-light;
+      cursor: pointer;
+      margin-left: -1rem;
+      margin-right: -1rem;
+      overflow: hidden;
+      padding: 1rem;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    .p-search-and-filter__clear {
```

Comment:
This doesn't seem to be anywhere in markup as well.

Reply:
Oh, I think both (prompt and clear) are from the state during typing into the component.

Looking at react example:
<img width="973" alt="Screenshot 2021-03-04 at 16 20 40" src="https://user-images.githubusercontent.com/83575/109986031-ac107400-7d05-11eb-91f6-90649a87bf6e.png">

We need either JS that adds those elements when typing (seems too much for the scope of the example) or a static example of the component in this state.

Reply:
Yeah I'm not sure about this, maybe we should chat tomorrow